utils/hare-shutdown

```diff
@@ -127,7 +133,7 @@ def exec_silent(cmd: str) -> bool:
 
 def exec(cmd: str) -> None:
     assert cmd
-    print('OK' if exec_silent(cmd) else '**ERROR**')
+    logging.info('OK' if exec_silent(cmd) else '**ERROR**')
```

Comment:
[Optional]: I think we don't need this ternary operator: if there is an **ERROR**, we don't need to `log.info()`, we should use `log.error()` instead